mobile/src/main/java/org/jraf/android/cinetoday/mobile/api/Api.java

```diff
@@ -101,6 +102,11 @@ public static Api get(Context context) {
                 .addQueryParameter(QUERY_DATE_KEY, SIMPLE_DATE_FORMAT.format(date))
                 .build();
         String jsonStr = call(url, false);
+        return getMovieList(jsonStr);
+    }
+
+    @VisibleForTesting
```

Comment:
I isolated the parsing of a json string into a separate method to be able to test parsing a json string from a test file, without having to mock a server.